apollo-portal/src/test/resources/sql/permission/insert-test-permissions.sql

```diff
@@ -13,5 +13,7 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
+ALTER TABLE Permission ALTER COLUMN DataChange_CreatedBy VARCHAR(255) NULL;
```

Comment:
are these alter statements still necessary as they are already defined in apollo-portal/src/test/resources/import.sql?

Reply:
fixed